src/lib/containers/table/StandaloneQueryWrapper.tsx

```diff
@@ -0,0 +1,155 @@
+import * as React from 'react'
```

Comment:
Moved this heavily used component from portals to SRC.  See https://github.com/Sage-Bionetworks/portals/blob/master/src/portal-components/StandaloneQueryWrapper.tsx